frontend/public/locales/en/translation.json

```diff
@@ -409,6 +410,18 @@
+    "largely_recommended": "<0>This is the default quality criterion of Tournesol 🌻.</0><1>It aims to measure the extent to which a content should be recommended to a very wide audience, according to the contributor's judgment.</1><2>The main motivation for making this the single default quality criterion is that it highlights the core of Tournesol's mission. Namely, Tournesol aims to identify contents worth promoting at scale.</2><3>A secondary motivation is to make it as contributor-friendly as possible, especially for new contributors.</3><4>A third motivation is to use this quality criterion to infer the importance assigned by the contributors the the other quality criteria. In particular, we plan to soon learn how to best combine the different optional quality criteria (and the default one) into a more reliable unique global Tournesol score per video.</4>",
+    "layman_friendly": "<0>How understandable is it, without prior knowledge?</0><1>This criterion aims to identify contents that would be suitable to those with very limited prior knowledge. When judging this feature, contributors are encouraged to reflect on the suitability of the content to their grandparents or to their 5-year-old nephew.</1><2>Contents that score high on \"layman-friendly\" should be accessible to a very large audience.</2><3>Distinction with \"clear and pedagogical\"</3><4>An example of layman-friendly non-pedagogical video is a video discussing concepts in a very superficial way.</4><5>An example of a very pedagogical video that may not be very layman-friendly is <2>this video by 3Blue1Brown</2>.</5>",
```

Comment:
> An example of a very pedagogical video that may not be very layman-friendly is <2>this video by 3Blue1Brown</2>.

In the future we could explain why this video of `3Blue1Brown` can be considered very pedagogical and not very layman-friendly. I think it's not pedagogical to let the users figure out by themselves why the video is not layman-friendly eh eh.